### PR TITLE
Admin Dashboard view of oEmbed enabled items

### LINKED ISSUE #76: Admin Dashboard view of oEmbed enabled items
### Descriptive summary

Admins and staff should be able to easily see only all works that have an oEmbed URL included.

### Expected behavior

An Admin Dashboard view can show which works have an oEmbed URL included.

### Related work

#75 Initial configuration of oEmbed

### PR DESCRIPTION
**Pull request by CGillen**
Fixes #76

### CODE REVIEW

File: app/controllers/hyrax/my_controller.rb.bak [hunk 1]
```diff
@@ -0,0 +1,67 @@
+module Hyrax
```
Comment (revgum) [inline app/controllers/hyrax/my_controller.rb.bak]: Don't forget to remove this file

**Comment by CGillen** [comment]
Current dashboard layout as of this commit:
image

- Title links to the work: `http://localhost:3000/concern/generics/pr76f340k?locale=en`
- oEmbed Link links to the visible url
- oEmbed Status is dummy placeholder text to swap out for logic of #77

**Comment by revgum** [comment]
Screenshot is looking good! Aside from the feedback you're asking from Ryan, this PR just needs tests to cover the code.

**Comment by wickr** [comment]
@CGillen No, the work URI doesn't need to be in a separate column. The link in the Title should be sufficient.

There shouldn't be more than 1 oEmbed link for any given work, so you can drop the bulleted list to simplify things.

**Comment by CGillen** [comment]
Then we should codify a single value relationship for the oEmbed URL. I can make that part of this PR for expedition.

**Pull request merged**